src/libm/CMakeLists.txt

```diff
@@ -189,23 +197,90 @@ endif()
 # Note: adds object file as sources via cmake conditional generator expression
 foreach(SIMD ${SLEEF_SUPPORTED_EXTENSIONS})
   if(COMPILER_SUPPORTS_${SIMD})
+    list(FIND EXT_ENABLE_ALIAS ${SIMD} INDEX_ALIAS)
+    string(TOLOWER ${SIMD} SIMDLC)
+
+    # SP functions
+
     # Create a library
-    add_library(${OBJECT_${SIMD}} OBJECT ${SIMD_SOURCES} ${HEADER_${SIMD}})    
-    target_compile_definitions(${OBJECT_${SIMD}} PRIVATE
-      ENABLE_${SIMD}=1
-      DORENAME=1
-      ${COMMON_TARGET_DEFINITIONS}
-      )
-    add_dependencies(${OBJECT_${SIMD}} rename${SIMD}.h.generated)
+    add_library(${OBJECTsp_${SIMD}} OBJECT sleefsimdsp.c ${HEADER_${SIMD}})    
+    if (INDEX_ALIAS EQUAL -1)
+      target_compile_definitions(${OBJECTsp_${SIMD}} PRIVATE
+	ENABLE_${SIMD}=1
+	DORENAME=1
+	${COMMON_TARGET_DEFINITIONS}
+	)
+    else()
+      add_custom_command(
+	OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/include/alias${SIMD}sp.h ${CMAKE_CURRENT_BINARY_DIR}/include/alias_${SIMDLC}_sp.h
+       	COMMENT "Generating alias_${SIMDLC}sp.h"
+       	COMMAND ${TARGET_MKALIAS} ${ALIAS_PARAMS_${SIMD}_SP} > ${CMAKE_CURRENT_BINARY_DIR}/include/alias_${SIMDLC}_sp.h
```

Comment:
Our internal CI are simulating the SVE version of the library, both with native aarch64 simulation of SVE and with X86 based tools. Because of the latter configuration,  I started cross compiling our downstream version on X86, targeting AArch64 Advanced SIMD and SVE. In such cases, it turned out that the `COMMAND` invocation of `add_custom_command` are not able to find the executable that are just referred via the target name, like in `COMMAND ${TARGET_MKALIAS} ...`. It turned out that the command should be specified using generator expression, as explained in https://stackoverflow.com/questions/4346281/path-to-target-output-file, resulting in  `COMMAND $<TARGET_FILE:${TARGET_MKALIAS} ...>`